web/client/observables/wps/common.js

```diff
@@ -6,29 +6,61 @@
   * LICENSE file in the root directory of this source tree.
   */
 
-const url = require('url');
-const {endsWith} = require('lodash');
-
-module.exports = {
-    getWPSURL: (urlToParse, options) => {
-        if (urlToParse) {
-            const parsed = url.parse(urlToParse, true);
-            let newPathname = parsed.pathname;
-            if (endsWith(parsed.pathname, "wfs") || endsWith(parsed.pathname, "wms")) {
-                newPathname = parsed.pathname.replace(/(wms|ows|wfs|wps)$/, "wps");
-            }
-            return url.format({
-                ...parsed,
-                search: null,
-                pathname: newPathname,
-                query: {
-                    service: "WPS",
-                    ...options,
-                    ...parsed.query
-                }
-            });
+import url from 'url';
```

Comment:
Can you document these methods to clarify their usage parameters etc ? It could be very useful for future usage.